washingMonitor.js

```diff
@@ -0,0 +1,346 @@
+function washingMonitorSensorCallback(){
+    washingMonitor.sensorCallbackFn();
+}
+
+(function (washingMonitor){
+
+    washingMonitor.sensorCallbackFn = null;
+    var monitor = null;
+
+    //TODO: Values should in caps, right?
+    washingMonitor.eventTypes = {
+        init: 0,
+        start: 1,
```

Comment:
I like them in camelCase here, in my mind PascalCase would be for C# or something similar.